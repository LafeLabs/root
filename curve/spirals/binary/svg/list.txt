svg1534812851.svg,svg1531522562.svg,