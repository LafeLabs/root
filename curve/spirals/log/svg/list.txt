svg1534812857.svg,