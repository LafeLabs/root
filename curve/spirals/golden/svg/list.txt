svg1534812843.svg,svg1534812836.svg,svg1532032772.svg,svg1531180313.svg,svg1531179612.svg,